docker-compose.yml

```diff
@@ -42,7 +42,7 @@ services:
 
   dbeaver:
     build: ./docker/dbeaver
-    container_name: dbeaver
+    container_name: dbeaver-${ENVIRONMENT_NAME}
```

Comment:
Should I move dbeaver to the reverse proxy compose file and add a config for accessing the testing and production database?